conan/test/utils/tools.py

```diff
@@ -451,6 +452,20 @@ def load_home(self, filename):
         except IOError:
             return None
 
+    def open(self, filename):
+        # CI is set in our GitHub Actions, so we don't open files there
+        if not os.environ.get("CI", False):
```

Comment:
Maybe better an assert/exception to force it failing if someone leaves a ``client.open()`` call in code?